easybuild/easyconfigs/l/LLVM/LLVM-3.8.1-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '3.8.1'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+builddependencies = [
+    ('binutils', '2.25'),
+    ('CMake', '3.6.1'),
+    # We use the minimal Python in GCCcore
+    ('Python', '2.7.12', '-bare'),
+]
+
+dependencies = [
+    ('ncurses', '6.0'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON'
+# required to install extra tools in bin/
+configopts += "-DLLVM_INSTALL_UTILS=ON"
+
+separate_build_dir = True
+
```

Comment:
@ocaisa please include a custom `sanity_check_paths` like we have in https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/l/LLVM/LLVM-3.8.0-intel-2016a.eb